package.json

```diff
@@ -55,12 +55,14 @@
   },
   "dependencies": {
     "@bpmn-io/extract-process-variables": "^0.4.1",
+    "@camunda/element-templates-json-schema": "0.2.0",
```

Comment:
If I understand it correctly, we want to update the version manually, right?

Reply:
Exactly